Makefile.include

```diff
@@ -30,8 +30,8 @@ include $(RIOTBASE)/Makefile.docker
 # Static code analysis tools provided by LLVM
 include $(RIOTBASE)/Makefile.scan-build
 
-# Path to the current directory relative to the git root
-BUILDRELPATH ?= $(shell git rev-parse --show-prefix)
+# Path to the current directory relative to RIOTBASE
+BUILDRELPATH ?= ${PWD:${RIOTBASE}/%=%}/
```

Comment:
The only thing different from the form described above is that I used curly braces instead of parens. Both versions seem to work for me, though.